src/v1beta2/cluster_controller_client.js

```diff
@@ -1041,9 +1047,13 @@ class ClusterControllerClient {
   }
 
   /**
-   * Gets cluster diagnostic information.
-   * After the operation completes, the Operation.response field
-   * contains `DiagnoseClusterOutputLocation`.
+   * Gets cluster diagnostic information. The returned
+   * Operation.metadata will be
+   * [ClusterOperationMetadata](https://cloud.google.com/dataproc/docs/reference/rpc/google.cloud.dataproc.v1beta2#clusteroperationmetadata).
+   * After the operation completes,
+   * Operation.response
+   * contains
+   * [Empty](https://cloud.google.comgoogle.protobuf.Empty).
```

Comment:
@bcoe @alexander-fenster looks like we have a bad link here.

Reply:
It's a bug in `gapic-generator`. We don't see it often because an LRO with `google.protobuf.Empty` as a result is a weird thing. Let's just move this library to TypeScript ASAP since the fix might be non-trivial and I don't want to touch gapic-generator code base anymore.  For now, a hack in `synth.py` can take care of this.

Reply:
Let's fix this link in `synth.py` and this will be fixed when we move to micro-generator.